.gitignore

```diff
@@ -7,3 +7,4 @@ package.xml
 composer.lock
 experiments/
 vendor/
+.idea
```

Comment:
work in progress. I have created the pull request to run the pipelines at travis!